phpBB/includes/mcp/mcp_post.php

```diff
@@ -197,7 +213,7 @@ function mcp_post_details($id, $mode, $action)
 		$l_deleted_by = '';
 	}
 
-	$template->assign_vars(array(
+	$mcp_post_template_data = array(
```

Comment:
Alternatively you could pick and choose what to send in the event, like for example $message, but you are right, mostly everything else is availaby by means of the $post_info array.